src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -349,62 +335,53 @@ In some cases adding content dynamically is an important part of user experience
+A number of CSS properties can be changed in a much more performant manner. For example, `transform` animations can be used to `translate`, `scale`, `rotate`, or `skew` without triggering a re-layout and so completely avoiding CLS.
 
-### Animations
+When animations are instead done but changing `top` and `left` instead of using `transform: translate` it causes CLS. This happens **even when the element being moved is in it's own layer and so does not cause shifts to other elements**. The net effect to the user may be the same, but one is counted as CLS, and one is not. This may seem like CLS should be changed to treat these the same, but there are other considerable performance benefits of using non-composited animations.
 
-**Summary:** Prefer `transform` animations to animations of properties that trigger layout changes.
+To learn more about what CSS properties trigger layout see [High-performance animations](/animations-guide/).
 
-Changes to CSS property values can require the browser to react to these changes. A number of values trigger re-layout, paint and composite such as `box-shadow` and `box-sizing`. A number of CSS properties can be changed in a less costly manner.
+### Web fonts
 
-To learn more about what CSS properties trigger layout see [High-performance animations](/animations-guide/).
+Downloading and rendering web fonts is typically handled in one of two ways before the web font is downloaded:
 
-### bfcache
+- The fallback font is swapped with web font (FOUT—flash of unstyled text)
+- "Invisible" text is displayed using the fallback font until a web font is available the text is made visible (FOIT—flash of invisible text)
 
-The back/forward cache, or bfcache keeps pages in browsers memory for a short period so if you return to them, then they will be restored exactly as you left them. This means the fully loaded page is instantly available—without any shifts which may be normally seen during load to to any of the reason above. When this was rolled out to Chrome, we saw [noticeable improvements in CLS](https://twitter.com/anniesullie/status/1491399685961293828?s=20&t=Qj_nzSRZD0_c-HaAnfr98Q).
+It is important to understand that **both of these cause CLS**. Even though the invisible is not visible to the user, it is layed out using the fallback font and so the text block of the font, and the surrounding content shifts when the web font loads—in the exact same way as for the visible font for FOUT.
 
-Some sites are ineligible for the bfcache due to a variety of reasons. Ensuring your site is eligible is a sure fire way to keep CLS scores down. Read [the bfcache article](/bfcache/) for more details on how to test and identify any issues preventing bfcache usage.
+The following tools can help you minimize this:
 
-### Developer Tools
+- `font-display: optional` can avoid a re-layout as the web font is only used if it is available by the time of initial layout.
```

Comment:
```suggestion
- `font-display: optional` can avoid a layout invalidation as the web font is only used if it is available by the time of initial layout.
```

Reply:
Let's keep this. We use "re-layout" in a few places in this doc (and did in the old version) and "invalidation" sounds kinda technical.